api/StorageEstimate.json

```diff
@@ -0,0 +1,148 @@
+{
+  "api": {
+    "StorageEstimate": {
```

Comment:
I have added the data despite the *Browser Compat Data migration* sheet stating that it is on hold due to #190.

As far as I can see, the [Storage Standard](https://storage.spec.whatwg.org/#idl-index) only requires a secure context for `NavigatorStorage` and `StorageManager`, but not for `StorageEstimate`. This make sense, because the `StorageEstimate` is only available via the `StorageManager` anyway (which requires a secure context).